connectors/filtering/validation.py

```diff
@@ -118,6 +124,20 @@ def to_s(cls, value):
             case FilteringValidationState.EDITED:
                 return "edited"
 
+    @classmethod
+    def from_string(cls, string):
+        match string.casefold():
+            case "valid":
+                return FilteringValidationState.VALID
```

Comment:
would it help to move

```
class FilteringValidationState(Enum):
    VALID = 1
    INVALID = 2
    EDITED = 3
```

to 

```
class FilteringValidationState(Enum):
    VALID = "valid"
    INVALID = "invalid"
    EDITED = "edited"
```

Reply:
Why did we use numbers in the first place for other enums? Your suggestion seems to be way more convenient. TIL

Reply:
> Why did we use numbers in the first place for other enums? 

because Enum are often used as a collection of integers, but that was a mistake to do so.

> Your suggestion seems to be way more convenient. 

Yeah and you can do stuff like this :

```
state = FilteringValidationState('valid')
```

Reply:
Cool, adjusted it. Thanks for creating the tech-debt issue